docs/visual-basic/language-reference/error-messages/handles-clause-requires-a-withevents-variable-defined.md

```diff
@@ -9,14 +9,42 @@ helpviewer_keywords:
 ms.assetid: 5b66f6a8-f050-4e03-a57f-a64e85f80cb5
 ---
 # Handles clause requires a WithEvents variable defined in the containing type or one of its base types
-You did not supply a `WithEvents` variable in your `Handles` clause. The `Handles` keyword at the end of a procedure declaration causes it to handle events raised by an object variable declared using the `WithEvents` keyword.  
+You did not supply a `WithEvents` variable in your `Handles` clause. The `Handles` keyword at the end of a procedure declaration causes it to handle events raised by an object variable declared using the `WithEvents` keyword.
   
- **Error ID:** BC30506  
-  
-## To correct this error  
+ **Error ID:** BC30506
+
+## To correct this error
   
-- Supply the necessary `WithEvents` variable.  
+- Supply the necessary `WithEvents` variable.
   
+## Example
+In the following example, we create a [System.Timers.Timer](https://docs.microsoft.com/dotnet/api/system.timers.timer) and use its `Elapsed` event to repeat some action. The error `BC30506` is produced when `WithEvents` isn't supplied to `_timer1` variable.
+
+```vb
+Imports System.Timers
+
+Module Module1
+    ' This line will cause an error in the Handles clause.
```

Comment:
I think @rpetrusha comment was that this line, which is critical to understanding the code, will not be localized in the docs (code isn't) 

I would suggest that we leave in this code sample the correct version (the declaration that includes "WithEvents" and have a separate code fragment with a paragraph (not code) that says "This would not compile in the code below." or something to that effect. 

@rpetrusha  can clarify as well.